web/assets/main.js

```diff
@@ -15,3 +15,25 @@ GOVUKFrontend.Tabs.prototype.setup = () => {};
 
 GOVUKFrontend.initAll();
 MOJFrontend.initAll();
+
+if (window.self !== window.top) {
+  document.body.className += " app-!-embedded";
```

Comment:
document.body.classList.add('app-!-embedded'); feels more explicit but feel free to ignore, not a biggy